src/modules/hyprland/workspaces.cpp

```diff
@@ -215,11 +326,19 @@ void Workspaces::sort_workspaces() {
                 return a->id() < b->id();
               }
               if (a->id() < 0 && b->id() < 0) {
-                if ((a->is_special()) ^ (a->is_special())) {
+                if ((a->is_special()) ^ (b->is_special())) {
                   return a->id() > b->id();
                 }
                 return a->id() < b->id();
               }
+              if ((a->id() == 0) || (b->id() == 0)) {
```

Comment:
what the order we are trying to go for here?

Reply:
Ok, I implemented the first option.